python/src/bxsolana_trader_proto.egg-info/PKG-INFO

```diff
@@ -0,0 +1,4 @@
+Metadata-Version: 2.1
```

Comment:
need a gitignore fo `.egg-info

Reply:
probably copy the .gitignore from `solana-trader-client-python` to `python` directory